week05/go.mod

```diff
@@ -0,0 +1,3 @@
+module week05
```

Comment:
Please remove the `week05/go.mod` file so your tests will run.

Reply:
ok

Reply:
@markbates I have removed the week05/go.mod and updated tests.yml but now the PR tests checks are failing. Please advise.